vault/census.go

```diff
@@ -2,5 +2,9 @@
 
 package vault
 
+import "time"
+
 // CensusAgent is a stub for OSS
```

Comment:
Should this comment be updated?

Reply:
This is still a stub, just the definition of "stub" has changed slightly 😄